libs/prov/prov_k8s_cortx_deploy.py

```diff
@@ -1790,3 +1794,43 @@ def namespace_name_generator(size):
         string_alpha = generated_string[:string_len] + "-" + generated_string[string_len:]
         LOGGER.info("The string is %s and length is %s", string_alpha, len(string_alpha))
         return string_alpha
+
+    def update_resource_limit_thirdparty(self, filepath):
+        """
+        This Method is used to update the resource limits for third party services
+        file: solution.yaml file
+        returns True
+        """
+
+        with open(filepath) as soln:
+            conf = yaml.safe_load(soln)
+            parent_key = conf['solution']  # Parent key
+            common = parent_key['common']
+            resource = common['resource_allocation']
+            consul = resource['consul']
+            zookeeper = resource['zookeeper']['resources']
+            kafka = resource['kafka']['resources']
+            type_list = ['requests', 'limits']
+            consul_list = ['server', 'client']
+            third_party_resource = self.deploy_cfg['thirdparty_resource']
+            # updating the consul server /client request and limit resources
+            for elem in consul_list:
+                for res_type in type_list:
```

Comment:
can we have type_list as outer for loop
Consul server,clients can be traversed in inner loop

Reply:
done